Formula/mono.rb

```diff
@@ -79,7 +78,6 @@ def install
     if build.with? "fsharp"
       resource("fsharp").stage do
         ENV.prepend_path "PKG_CONFIG_PATH", lib/"pkgconfig"
-        system "./autogen.sh", "--prefix=#{prefix}"
```

Comment:
If we're removing this line can we remove `depends_on "automake" => :build` et al.?

Reply:
No, mono depends on automake.

Reply:
Still? Those dependencies were only added when `fsharp` was back in https://github.com/Homebrew/homebrew-core/commit/949ae5c935b.

Reply:
Hmm, I see mono in brew just call ./configure without autogen.sh. I'll try to remove it.